app/assets/stylesheets/components/_address.scss

```diff
@@ -18,11 +18,11 @@
   line-height: 2;
   color: $color-link-blue;
 
-  &:hover {
+  &:hover, &:focus {
     text-decoration: none;
```

Comment:
Yep — good question. It's applied on the `.value` element below. Here's `.address_link` in red, and `.address_link .value` in blue:

![image](https://cloud.githubusercontent.com/assets/1007202/7587567/368b1a36-f8af-11e4-9e51-1a32d2ab4a8d.png)